client/payment-methods-map.js

```diff
@@ -85,10 +85,11 @@ const paymentMethodsMap = {
 		label: __( 'Affirm', 'woocommerce-gateway-stripe' ),
 		// translators: %s is the store currency.
```

Comment:
nit: This is not an issue in this PR but we'd want to update the `translators` comment as it seems outdated.